temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt

```diff
@@ -205,7 +227,12 @@ fun AmountInput(
         forCalculator = false,
         onNumberPressed = {
             if (firstInput) {
-                setAmount(it)
+                if (amount == "-0") {
```

Comment:
We should prevent "-0" from ever happening

Reply:
I was going to ask for a confirmation on that one since the issue requested to include "-0"'s case as well. I can drop that case and make it so that it only works for non-empty values.

Reply:
Yes, just drop "-0" and it would be good